services/brig/deb/opt/brig/templates/team/en/email/new-team.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Wire</title>
+</head>
+
+<body style="
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5em;
+  color: #42474A;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+">
+  <br/>
+  <table width="100%" cellpadding="0" cellspacing="0">
+    <tr>
+      <td>
+        <center>
+          <table cellpadding="0" cellspacing="10" style="
+            max-width: 500px;
+            margin: 12px auto 0 auto;
+            font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif;
+          ">
+            <thead>
+              <tr>
+                <td style="padding-bottom: 0px;">
+                  <a href="https://wire.com">
+                    <img src="https://wire.com/p/img/email/logo-email-black.png" width="57" height="18" alt="Wire" />
+                  </a>
+                </td>
+                <td style="text-align: right; text-transform: uppercase; font-size: 12px; font-weight: 500;">
+                  <a href="https://wire.com" style="text-decoration: none; white-space: nowrap; color: #42474A;">wire.com</a>
+                </td>
+              </tr>
+            </thead>
+            <tbody style="font-size: 16px;">
+              <tr>
+                <td width="500" colspan="2" style="padding-top: 48px; padding-bottom: 0px;">
+
+                  <h1 style="font-size: 32px; font-weight: 300; text-align: center; line-height: 1.3;">
+                    Your new team
+                  </h1>
+                  <p style="color: #42474A; margin-top: 48px;">
+                    You have just created a team called
+                    <strong>{{teamName}}</strong> on Wire with this email address. Your 30–day free trial starts today.
```

Comment:
`${teamName}`
This is meant to be sent _after_ an owner has activated a team account, right?